frontend/src/components/layout/Layout.tsx

```diff
@@ -1,4 +1,4 @@
-import { ReactNode } from "react";
+import React, { ReactElement, ReactNode } from "react";
```

Comment:
No biggie, but this is an unused import now I think.

```suggestion
import { ReactElement, ReactNode } from "react";
```